issue_release_status.go

```diff
@@ -88,7 +88,17 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 	prNum := event.GetNumber()
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
 
-	ctx, logger := githubapp.PreparePRContext(ctx, installationID, repository, prNum)
+	logctx := zerolog.Ctx(ctx).With()
+	logctx = logctx.Int64("github_installation_id", installationID)
+	logctx = logctx.Str("github_repository_owner", repository.GetOwner().GetLogin())
+	logctx = logctx.Str("github_repository_name", repository.GetName())
+	logctx = logctx.Int("github_pr_num", prNum)
+
+	logger := logctx.Logger()
+	ctx = logger.WithContext(ctx)
+	//ctx, logger := githubapp.PreparePRContext(ctx, installationID, repository, prNum)
```

Comment:
Can we drop this?

Reply:
Yea

Reply:
https://github.com/lunarway/release-manager-bot/pull/21/commits/82491778f9815f68585888e12d47c201c59644d7